src/lib/model/HubCertificateModel.js

```diff
@@ -26,45 +26,30 @@ class HubCertificateModel {
      * Gets Hub CA list from MCM Server
      *
      * @param opts {object}
-     * @param opts.envId {string}
      * @param opts.dfspId {string}
      */
-    async getHubCAS(opts) {
-        const url = `/environments/${opts.envId}/hub/cas`;
-        return this._mcmServerRequest.get(url);
-    }
-
-    /**
-     * Gets Hub CA from MCM Server
-     *
-     * @param opts {object}
-     * @param opts.envId {string}
-     * @param opts.dfspId {string}
-     */
-    async getRootHubCA(opts) {
-        const url = `/environments/${opts.envId}/ca/rootCert`;
+    async getHubCA(opts) {
+        const url = `/hub/ca`;
```

Comment:
As it's not mentioned in the discussion, I just want to check that this change is intended to be included in the PR?

Reply:
Yes, it's used in `mojaloop-payment-manager-management-api`